dev/dump.sql

```diff
@@ -67,10 +68,11 @@ CREATE VIEW public.account_current_interest_view AS
 
 CREATE TABLE public.accounts (
     account_id uuid DEFAULT gen_random_uuid() NOT NULL,
-    email character varying NOT NULL,
+    email character varying,
     created_at timestamp without time zone DEFAULT now() NOT NULL,
     status character varying NOT NULL,
-    source character varying NOT NULL
+    source character varying NOT NULL,
+    is_deleted boolean DEFAULT false NOT NULL
```

Comment:
These look like reasonable changes to the database schema but they'll need to be applied in a different place. We use `alembic` to manage changes to the schema as a series of migrations—here's [an example](https://github.com/CCRI-POPROX/poprox-storage/blob/main/poprox-db/migrations/versions/2024_09_11_1524-f7953f878964_add_treatment_id_to_newsletters_table.py) of one that adds a column to an existing table. You can create a new migration from the `poprox-db` directory with `alembic revision -m "a description of the change"`

Reply:
(This file is exported from our database as a way to provide some data that can be loaded for testing, but I think it may be out of date and/or missing some tables.)

Reply:
(to be clear, it is out of date, but we also run alembic heads on top of this to get the tables set up correctly on init so things should largely be OK regardless...)